SocketRocket/SRWebSocket.m

```diff
@@ -534,6 +534,14 @@ - (void)_initializeStreams;
     
     CFStreamCreatePairWithSocketToHost(NULL, (__bridge CFStringRef)host, port, &readStream, &writeStream);
     
+    //If voip is set in UIBackgroundMode, set apropriate properties on read and write stream
+    NSArray* backgroundModes = [[[NSBundle mainBundle] infoDictionary] objectForKey:@"UIBackgroundModes"];
+    
+    if (backgroundModes && [backgroundModes containsObject:@"voip"]) {
+        CFReadStreamSetProperty(readStream, kCFStreamNetworkServiceType, kCFStreamNetworkServiceTypeVoIP);
```

Comment:
`kCFStreamNetworkServiceTypeVoIP` has been deprecated in iOS 9, which might make this PR a non-starter. Again, I'll defer to @mikelikespie